.azure-pipelines/buildAndPackage.yml

```diff
@@ -0,0 +1,36 @@
+#Copyright (c) Microsoft Corporation. All rights reserved.
+#Licensed under the MIT License.
+#Building and packaging the artifacts of the Java-SDK libraries using the build.gradle file.
+#Ready the package for deployment and release. 
+
+trigger:
+  branches:
+    include:
+    - dev 
+    - main
+    - feature/v3
+  paths: 
+    include:
+      - src/*
+    exclude:
+      - docs/*
+      - .github/*
+      - LICENSE
+      - .gitignore
+      - README-localized/*
+      - CONTRIBUTING.md
+      - THIRD PARTY NOTICES
+      - README.md
+
+pr: none
+
+pool:
+  vmImage: ubuntu-latest
+
+steps: 
```

Comment:
This currently only defines the build stage. A multi-stages pipeline should be defined to also contain the CD steps.
It should leverage environments and approvals
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/approvals?view=azure-devops&tabs=check-pass
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/environments?view=azure-devops
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#stage